src/components/home/Empower.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Title from "@/components/Title";
+
+const Empower = () => {
+  return (
+    <div className="flex items-center justify-center flex-col">
```

Comment:
great job Mailyn, this looks almost ok to merge. Just need to use rows and cols here instead of div. Check out how Kelly did her issue for introduction.js. Lmk of questions